.github/scripts/generate_binary_build_matrix.py

```diff
@@ -415,6 +413,26 @@ def generate_wheels_matrix(
                             ),
                         }
                     )
+                    # special build building  PyThon 3.11 for 12.1 CUDA
+                    if python_version == "3.11" and arch_version == "12.1":
```

Comment:
Let's mention that this is for colab only. And also, we need 3.10 for this one, aren't we?